packages/dom-utils/.eslintrc.json

```
@@ -0,0 +1,6 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "rules": {
+    "@typescript-eslint/consistent-type-assertions": "off" // 1 error
```

For one error, does it make sense to use an inline comment to ignore the error?